File: 00_조준장/로또의 최고 순위와 최저 순위.kt
class Solution {
    fun solution(lottos: IntArray, win_nums: IntArray): IntArray {
        var answer: IntArray = intArrayOf(lottos.count { it == 0 }, 6)
        val rank = intArrayOf(6, 6, 5, 4, 3, 2, 1)

        win_nums.forEach { winNum ->
            if (winNum in lottos) answer[0] += 1
            else answer[1] -= 1

        }
        answer = intArrayOf(rank[answer[0]], rank[answer[1]])

        return answer
    }
}

File: 08_이훈일/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    rank={6:1,5:2,4:3,3:4,2:5,1:6,0:6}
    answer = []
    sameCount=0

    for i in lottos:
        if i in win_nums:
            sameCount+=1
            
    zeroCount=lottos.count(0)
    answer.append(rank[zeroCount+sameCount])
    answer.append(rank[sameCount])
    return answer
